projects/public-search/src/app/document-brief/document-brief.component.html

```diff
@@ -24,7 +24,7 @@
     </div>
     <div class="card-body w-100 py-0">
       <h4 class="card-title mb-1">
-        <a target="_self" href="/{{ detailUrl }}">{{ record.metadata.title }}</a>
+        <a target="_self" href="/{{ detailUrl.link }}">{{ record.metadata.title }}</a>
```

Comment:
Remove href tag and add `[routerLink]="[detailUrl.link]"`

Reply:
Done